drafts/personal-security-checklist.md

```diff
@@ -6,20 +6,44 @@
 - [ ] No-Wifi Printer: Not an allowed expense
 - [ ] Hardware Wallet: Allowed to expense 1 per contributor
 
-
 ## Personal Procedures
+- [ ] Join internal incident channel
+    - [ ] turn on notifications
+
+### PC      
 - [ ] Get rid of clipboard managers
     - Reasoning [here](https://coinjournal.net/pc-malware-steals-funds-modifying-ethereum-addresses/)
 - [ ] Get rid of cloud screenshot auto-upload
 - [ ] Get rid of remote viewer
-- [ ] Get a password manager
+- [ ] Get a password manager (default: bitwarden)
     - [ ] secure with 2FA (YubiKey / Google Auth)
-- [ ] Remove unused browser extensions
-- [ ] Disable infrequently used browser extensions
+- [ ] Disable unused/infrequently used browser extensions
 - [ ] If your computer is old, start fresh
     - [ ] fresh OS install
     - [ ] new computer
+- [ ] Do not tamper with the integrity protection of your OS (OSX SIP, Linux App Armor)
+    - [ ] Bonus: Get a vm and use put your browser and mail client in the vm.
 - [ ] Review launch-on-startup software
+
+### Mobile
+- [ ] Get Trail of Bits iVerify.
+    - [ ] Go through the tutorials and change the settings accordingly.
+- [ ] Do not charge your phone with a charger that is not yours/from someone you trust.
+- [ ] Get a phone that supports yubikey plugable/nfc
+- [ ] Get a vpn for your phone. Nordvpn works.
```

Comment:
why specify Nord?

Reply:
Not a specification, just as recommandation if folks don't wanna go through the effort of researching their own thing but sure questionable